examples/waiting-for-goroutines-to-finish/waiting-for-goroutines-to-finish.go

```diff
@@ -0,0 +1,47 @@
+// To wait for multiple goroutines to finish, we can
+// use a sync.WaitGroup.
```

Comment:
By convention we italic important terms when we first introduce them in an example.

Reply:
We usually use the plain-English word(s) when introducing a concept and throughout the example when we refer to it in the text. For example in /timers we introduce "timers" and refer to the "timer" not `time.Timer`. WaitGroup is a little less obvious since it's not a standard CS phrase (I don't think?). But e.g. "we can use a _wait group_ ..." and "`wg` is the wait group it uses ..." doesn't seem obviously right either. I'm not sure.

Reply:
Changed heading comment, refactored the other comment.